services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImplV2.java

```diff
@@ -167,4 +228,91 @@ public void unarchiveNews(String newsId, String currentUserName) throws Exceptio
   public boolean canArchiveNews(Identity currentIdentity, String newsAuthor) {
     return false;
   }
+
+  public News createDraftArticleForNewPage(News news, String pageOwnerId) throws Exception {
+    Page rootPage = getArticlesRootPage(pageOwnerId);
+    if (rootPage != null) {
+      DraftPage draftArticle = new DraftPage();
+      draftArticle.setNewPage(true);
+      draftArticle.setTargetPageId(null);
+      draftArticle.setTitle(news.getTitle());
+      draftArticle.setContent(news.getBody());
+      draftArticle.setParentPageId(rootPage.getId());
+      draftArticle.setAuthor(news.getAuthor());
+      draftArticle.setActivityId(news.getActivityId());
+      // created and updated date set by default during the draft creation process
+      draftArticle = noteService.createDraftForNewPage(draftArticle, System.currentTimeMillis());
+      if (draftArticle != null) {
+        // save illustration
+        Long illustrationId = -1L;
+        if (news.getUploadId() != null && !news.getUploadId().isBlank()) {
+          illustrationId = saveNewsIllustration(news.getUploadId(), null);
+        }
+        Map<String, String> properties = new HashMap<>();
+        properties.put("summary", news.getSummary());
+        properties.put("publicationState", news.getPublicationState());
+        properties.put("uploadId", news.getUploadId());
+        properties.put("illustrationId", illustrationId >= 0 ? String.valueOf(illustrationId) : null);
+        NewsDraftObject newsDraftMetaDataObject =
+                                                new NewsDraftObject(NEWS_METADATA_DRAFT_OBJECT_TYPE, draftArticle.getId(), null);
+        MetadataKey newsDraftMetadataKey = new MetadataKey(METADATA_TYPE.getName(), "news", 0);
```

Comment:
```suggestion
        MetadataKey newsDraftMetadataKey = new MetadataKey(NEWS_METADATA_TYPE.getName(), "news", 0);
```
define a constant NEWS_METADATA_NAME for "news"